kubernetes/controller/internal/setup/repository.go

```diff
@@ -0,0 +1,53 @@
+package setup
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+
+	"ocm.software/open-component-model/bindings/go/credentials"
+	"ocm.software/open-component-model/bindings/go/plugin/manager"
+	"ocm.software/open-component-model/bindings/go/repository"
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+// RepositoryOptions configures OCM repository creation.
+type RepositoryOptions struct {
+	PluginManager   *manager.PluginManager
```

Comment:
imo the plugin manager is not an option, but mandatory. WDYT?

Reply:
Then yeah. It shouldn't. :D